src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/Connection/TransportCreationSubHandlers/IpAddressVersionComparer.cs

```diff
@@ -7,33 +7,33 @@
 using System.Net;
 using System.Net.Sockets;
 
-namespace Microsoft.Data.SqlClientX.Handlers.TransportCreation
+namespace Microsoft.Data.SqlClientX.Handlers.Connection.TransportCreationSubHandlers
```

Comment:
Handlers need not come twice in namespace, would recommend naming as: "Microsoft.Data.SqlClientX.Handlers.Connection.TransportCreation"

And that it also contains the TransportCreationHandler.cs class.

Similar should be done for PreLoginHandler.cs, can be done separately.